hack/scale_out_poc/two_TP/nginx.conf

```diff
@@ -0,0 +1,119 @@
+user www-data;
```

Comment:
delete this file and all the other nginx files.

Reply:
deleted